governance/votes/Executive vote - November 19, 2021.md

```diff
@@ -0,0 +1,168 @@
+
+| VAULT TYPE | CURRENT LLL | PROPOSED LLL | LLL DIFFERENCE |
+|--|--|--|--|
+| ETH-A | 40 million DAI | 65 million DAI | +25 million DAI |
+| ETH-B | 25 million DAI | 30 million DAI | +5 million DAI |
+| ETH-C | 30 million DAI | 35 million DAI | +5 million DAI |
+
+### A change of Covenants for P1-DROP (Peoples Company Series 1)
+
+As per [this poll], the following covenant changes to P1-DROP (Peoples Company Series 1) will occur if this executive proposal passes:
+
+- Increase maximum acquisition price from $1.5 million to $2.85 million.
+- Include Arkansas as a possible state.
+
+### Offboarding Vaults
+
+The listed vaults will be offboarded by implementing the following changes if this executive proposal passes:
+
+#### BAL-A
+
+As per [this poll](https://vote.makerdao.com/polling/QmcwtUau), the following parameters changes will occur to offboard BAL-A if this executive proposal passes:
+
+- Lower the [Liquidation Penalty (`chop`)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+- Lower the [Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+- Increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) from 165% to 2000% over 30 days using dss-lerp.
+
+#### COMP-A
+
+As per [this poll](https://vote.makerdao.com/polling/QmRDeGCn), the following parameters changes will occur to offboard COMP-A if this executive proposal passes:
+
+- Lower the [Liquidation Penalty (`chop`)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+- Lower the [Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+- Increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) from 165% to 2300% over 30 days using dss-lerp.
+
+#### AAVE-A
+
+As per [this poll](https://vote.makerdao.com/polling/QmPdvqZg), the following parameters changes will occur to offboard AAVE-A if this executive proposal passes:
+
+- Lower the [Liquidation Penalty (`chop`)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+- Lower the [Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+- Increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) from 165% to 2100% over 30 days using dss-lerp.
+
+#### LRC-A
+
+As per [this signal poll](https://forum.makerdao.com/t/signal-request-offboard-unprofitable-collateral-types/8628), the following parameters changes will occur to offboard LRC-A if this executive proposal passes:
+
+- Cancel previous lerp contract and increase the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) to 24300% over 30 days using dss-lerp.
```

Comment:
Would split these into two:
- The previous dsslerp for increasing the [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) of this collateral will be canceled.
- A new dsslerp contract will be deployed to increase the liquidation ratio to 24,300% over 30 days.